### PR TITLE
Fix vis not reducing in height on resize

### PR DESCRIPTION
**Pull request by axelboc**

### CODE REVIEW

**Review by loichuder — CHANGES_REQUESTED** [review]
I tried doing something similar when I implemented the horizontal colorbar. 

But fixing the display by adding a decorator only to the `Horizontal` story means it does not work when switching from vertical to horizontal in other ColorBar stories :slightly_frowning_face:

File: .storybook/decorators/FillHeight.tsx [hunk 1]
```diff
@@ -1,10 +1,10 @@
 import React from 'react';
 import type { Story } from '@storybook/react';
 
-function FillHeight(VisCanvasStory: Story) {
+function FillHeight(WrappedStory: Story) {
```
Comment (loichuder) [inline .storybook/decorators/FillHeight.tsx:4]: `aStory` ?

**Review by loichuder — APPROVED** [review]

File: .storybook/decorators/FillHeight.tsx (hunk 1 above)
Comment (axelboc) [reply .storybook/decorators/FillHeight.tsx:4]: Needs to have a capital letter because it's a component.

Comment (loichuder) [reply .storybook/decorators/FillHeight.tsx:4]: `MyStory` ? It is just that I don't like `WrappedStory` (wrapped by whom ? by what ?)

Comment (axelboc) [reply .storybook/decorators/FillHeight.tsx:4]: It's what they use for the HOC pattern: https://reactjs.org/docs/higher-order-components.html, but I hesitated with `MyStory`, so I'm fine with that too 👍

**Pull request merged**